apps/production/prod-rucio-daemons.yaml

```diff
@@ -11,6 +11,15 @@ conveyorThrottlerCount: 2
 judgeEvaluatorCount: 2
 tracerKronosCount: 3
 judgeCleanerCount: 2
+replicaRecovererCount: 1
+minosTemporaryExpirationCount: 1
+
+replicaRecoverer:
+  activeMode: "false"
+  secretMounts:
+    - secretFullName: replica-recoverer-config
```

Comment:
Have these secrets been injected into the cluster?
I do not see them.

Reply:
I was planning to add all configs after this PR is merged, including the secret

Reply:
I think you should do it before.

Reply:
I've just created it 
```
$ k get secrets | grep repl
replica-recoverer-config                                   Opaque               1      14s
```